lib/generator/command_line.rb

```diff
@@ -95,8 +95,8 @@ def option_parser
     end
 
     def usage
-      option_parser.help +
-        "\nAvailable exercise generators:\n" +
+      option_parser.help <<
```

Comment:
Quick safety check on the value that is returned from the `help` method.  The `<<` that I was referencing was the second concatenation, rather than both.  The reason is I did not peek at the help method, to make sure that modifying that in place would not cause a change that affects other calls to help.